website/docs/r/azure_ai_services.html.markdown

```diff
@@ -0,0 +1,150 @@
+---
+subcategory: "Cognitive Services"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_azure_ai_services"
+description: |-
+  Manages Azure AI Services.
+---
+
+# azurerm_azure_ai_services
+
+Manages Azure AI Services.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_azure_ai_services" "example" {
+  name                = "example-account"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku_name            = "S0"
+
+  tags = {
+    Acceptance = "Test"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Azure AI Services. Changing this forces a new resource to be created.
```

Comment:
Please update all references in the doc to use `AI Services Account` instead of `Azure AI Services`
```suggestion
* `name` - (Required) Specifies the name of the AI Services Account. Changing this forces a new resource to be created.
```